app/src/main/res/xml/preferences.xml

```diff
@@ -6,18 +6,16 @@
 
         <EditTextPreference
             android:key="pref_video_host"
-            android:title="@string/host_settings_title"
-            android:summary="%s"/>
+            android:title="@string/host_settings_title" />
 
         <EditTextPreference
             android:key="pref_video_port"
-            android:title="@string/port_settings_title"
-            android:summary="%s"/>
+            android:numeric="integer"
+            android:title="@string/port_settings_title" />
 
         <EditTextPreference
             android:key="pref_video_password"
-            android:title="Password"
-            android:summary="%s"/>
+            android:title="Password" />
```

Comment:
Since you are touching it, could you please make it a password (i.e. masked with `*`) field?